src/Arcus.WebApi.Correlation/CorrelationOptionsOperation.cs

```diff
@@ -0,0 +1,33 @@
+﻿using GuardNet;
+
+namespace Arcus.WebApi.Correlation
+{
+    /// <summary>
+    /// Correlation options specific for the operation ID.
+    /// </summary>
+    public class CorrelationOptionsOperation
+    {
+        private string _headerName = "X-Operation-ID";
```

Comment:
Should be `RequestId`
```suggestion
        private string _headerName = "RequestId";
```

Reply:
Done.